### DISCUSSION TITLE
"The Hubble Constant: A Historical Review"

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#16] [Redshift Models]
Han, you write:

> [The Hubble Constant: A Historical Review](https://arxiv.org/abs/2305.11950) . Thanks for that reference in the first place.
I didn't know that an **accuracy of 1%** is claimed for the Hubble parameter using *the current preferred ΛCDM cosmological model*.
*Best values of* $H_0$ *from the distance ladder lie in the range 73 - 75 km/s/Mpc*, which may be much less accurate, but better physics I think.
In (not really) my own theory, there are two (mathemacist :-) approaches to the Hubble tension.
The first one is expressed by $H = 2/t$. The second one is expressed by  $H = \sqrt{\chi G \rho}$ , where $\chi = 8 \pi/3$ if according to ΛCDM.
Symbols are: $t =$ age of "the universe", $G =$ gravitational constant, $\rho =$ "density" of mass in "the universe".
I guess your favorite @marmetl is not among these two. Whatever, allow me to proceed along these lines anyway.

> In the [G. de Vaucouleurs](http://www.alternatievewiskunde.nl/cosmology/helaas.htm) section at my website we find, instead of the $\chi = 8 \pi/3$ value according to ΛCDM, a slightly greater value.
Namely: $\chi = 8 \pi \times 0.3883945571$.
Let us suppose that ΛCDM is employing the wrong value $(8 \pi/3)$ for that mathematical constant $\chi$.
And suppose that it must be the latter instead, according to UAC: $\chi = 8 \pi \times 0.3883945571$.
Then we get: $\boldsymbol{H :=} \sqrt{0.3883945571 / (1/3)} \times 67.4 =$ **72.7**5403614, with 1% accuracy.
*Best values of* $H_0$ *from the distance ladder lie in the range 73 - 75 km/s/Mpc*. (Coincidence, of course :-)

  Coincidence, of course!  Here's why.

  Historically, the published values of $H_0$ based on the distance-ladder went as low as [50 km/s/Mpc in the 1950s](https://ui.adsabs.harvard.edu/abs/1996PASP..108.1073T/abstract) before increasing to $\approx 70$ km/s/Mpc in 2000.

  The CMB-based value of $H_0$ was found by **selecting**, *out of **several** cosmologies*, the model that gave the nearest value to the distance-ladder measurements.  So in the 1990s cosmologists *took a snapshot* of the Hubble constant from distance-ladder measurements, $H_0 \approx 67 \pm 10$ km/s/Mpc at the time, and imbedded it in ΛCDM's [parameters](https://en.wikipedia.org/wiki/Lambda-CDM_model#Parameters).

  Riess *et al.* improved distance-ladder measurements which eventually gave $H_0 \approx 73.3$ km/s/Mpc, with a low uncertainty.  At the same time, refined measurements of the CMB were analyzed using 1990s-derived parameters, and unsurprisingly obtained $H_0 \approx 67.4$ km/s/Mpc with a low uncertainty.

  The ratio you are considering, $72.75/67.4$, is the ratio of historical values of the distance-ladder measured $H_0$ , one in 2022, the other in 1997.  The difference between these values depends on a random statistical uncertainty of measurements.
Since it's a random number, it's a coincidence that it matches $\sqrt{0.3884 / (1/3)}$, of course!

<img src="https://lweb.cfa.harvard.edu/~dfabricant/huchra/hubble/ho.2004.jpg">

From [THE HUBBLE CONSTANT](https://lweb.cfa.harvard.edu/~dfabricant/huchra/hubble/) by John P. Huchra.

**Comment by RedshiftDrift** [comment]
Original article:
by R. Brent Tully [arXiv:2305.11950](https://arxiv.org/abs/2305.11950)

"As the 20th century came to an end, ladder measurements of the Hubble constant were at odds with the favored cosmological model of the time of cold dark matter with Λ = 0. The new favorite became the ΛCDM model with dark energy giving rise to acceleration of space in a topologically flat universe. Yet ladder measurements, continuously improving, create doubts that this currently favorite model is complete. Yes, there is a Hubble tension."

**Reply by HanDeBruijn** [reply]
Here I am, Louis. Thanks for formatting my response to ACG mail properly. I will do it myself next time.
Have a good day. Good answer anyway.

Han
